extensions/BOP/twitch.js

```diff
@@ -0,0 +1,324 @@
+            text: "close chat",
+            blockType: Scratch.BlockType.COMMAND,
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Overlays",
+          },
+          {
+            opcode: "text",
+            text: "create text overlay with text [text] with italic [italic] with bold [bold] with size [size] with font [font]",
+            blockType: Scratch.BlockType.COMMAND,
+            arguments: {
+              text: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "hi chat",
+              },
+              italic: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              bold: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              size: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "20",
+              },
+              font: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "font",
+              },
+            },
+          },
+          {
+            opcode: "image",
+            text: "create image with image [image] with width [width] with height [height] with position [position] is bottom [bottom] is top [top]",
+            blockType: Scratch.BlockType.COMMAND,
+            arguments: {
+              image: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "https://tekinical.github.io/twitchURI.png",
+              },
+              width: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "50",
+              },
+              height: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "50",
+              },
+              position: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "position",
+              },
+              bottom: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              top: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+            },
+          },
+          {
+            opcode: "closetext",
+            text: "close text",
+            blockType: Scratch.BlockType.COMMAND,
+          },
+          {
+            opcode: "closeimage",
+            text: "close image",
+            blockType: Scratch.BlockType.COMMAND,
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Utilities",
+          },
+          {
+            opcode: "true",
+            text: "true",
+            blockType: Scratch.BlockType.BOOLEAN,
+          },
+          {
+            opcode: "false",
+            text: "false",
+            blockType: Scratch.BlockType.BOOLEAN,
+          },
+        ],
+        menus: {
```

Comment:
You can let all of these menus accept reporters

Reply:
What do you mean by this?

Reply:
set the "acceptReporters" value to true

Reply:
complete